public/index.html

```diff
@@ -32,8 +32,13 @@ <h1>JASON JOHN</h1>
         <a id="openModalButton" href="https://withjoy.com/jason-and-esha/rsvp" target="_blank">RSVP HERE</a>
 
     </div>
+    <!-- Info Button -->
```

Comment:
I just didn't like it being at the bottom of this div, but it actually appears on top in the actual page. Feel free to move it back if needed.